src/content/post/2023/02/21/approve-private-endpoint-connections-with-bicep.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Approve Private Endpoint Connections"
+date: 2023-02-21T18:32:41Z
+subtitle: "With Bicep"
+image: "/content/approve-private-endpoint-connections-with-bicep/taylor-vick-M5tzZtFCOfs-unsplash.jpg"
+summary: "Managing private endpoint connections - especially across Azure AD tenants - can be a chore. In many cases you will have to do this manually and often you will need elevated permissions. But usually your deployment pipeline already has sufficient permissions on the resource to approve the endpoint connection, so it would be much more convenient if we could make the approval a part of our desired state config."
+tags: ["Azure", "AKS", "Kubernetes", "Private Link", "Private Endpoint"]
+series: ["Private Link"]
+---
+
+Managing private endpoint connections - especially across Azure AD tenants - can be a chore. In many cases you will have to do this manually and often you will need elevated permissions. But usually your deployment pipeline already has sufficient permissions on the resource to approve the endpoint connection, so it would be much more convenient if we could make the approval a part of our desired state config.
+
+### Approve a private endpoint with Bicep
+
+It is actually very simple to approve a private endpoint using Bicep because the private endpoint connection - which is the link between the private endpoint and the target resource - is a child resource of the target resource. 
```

Comment:
It is actually very simple to approve a private endpoint using Bicep because **of** the private endpoint connection - I dont like the - maybe rewrite it together something like
It serves as the connection between the target resource and the private endpoint and is a child resource of the target resource.
?